include/bluetooth/nrf/host_extensions.h

```diff
@@ -107,6 +107,15 @@ struct bt_conn_set_pcr_params {
  */
 int bt_conn_set_power_control_request_params(struct bt_conn_set_pcr_params *params);
 
+/** @brief Reduce the priority of the initiator when following AUX packets.
+ *
+ *  @param reduce Set to true to reduce the priority. Set to false to restore the default priority.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @retval -ENOBUFS HCI command buffer is not available.
+ */
+int bt_host_extension_reduce_initator_aux_channel_priority(bool reduce);
```

Comment:
I have a slight pref for `bt_sdc_reduce_initiator_aux_channel_priority()` or `bt_nrf_extensions_reduce_initiator_aux_channel_priority()` if we want to kinda keep the namespace/path of the file

Reply:
`bt_nrf_host_extension_sdc_*` ?

Reply:
will remove the sdc part from the name.